Content.Shared/Actions/ActionContainerSystem.cs

```diff
@@ -26,6 +30,24 @@ public override void Initialize()
         SubscribeLocalEvent<ActionsContainerComponent, ComponentShutdown>(OnShutdown);
         SubscribeLocalEvent<ActionsContainerComponent, EntRemovedFromContainerMessage>(OnEntityRemoved);
         SubscribeLocalEvent<ActionsContainerComponent, EntInsertedIntoContainerMessage>(OnEntityInserted);
+        SubscribeLocalEvent<ActionsContainerComponent, MindAddedMessage>(OnMindAdded);
+        SubscribeLocalEvent<ActionsContainerComponent, MindRemovedMessage>(OnMindRemoved);
+    }
+
+    private void OnMindAdded(EntityUid uid, ActionsContainerComponent component, MindAddedMessage args)
+    {
+        if(!_mind.TryGetMind(uid, out var mindId, out var mindComp))
+            return;
+
+        mindComp.MindActionsContainer ??= EnsureComp<ActionsContainerComponent>(mindId);
```

Comment:
Just do try get comp

Reply:
TryGetMind works out better because mindComp.OwnedEntity can be null so this takes care of 2 checks at once.

Reply:
I meant to get the MindActionsContainer